ui/components/multichain/app-footer/__snapshots__/app-footer.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`App Footer should match snapshot 1`] = `<div />`;
```

Comment:
Oh oh are we missing something here to render the full DOM?

Reply:
Updated! This was because of the conditional rendering https://github.com/MetaMask/metamask-extension/pull/20751/commits/4c2098d8157cbfd02ce755b58e88f977db6b6afe